packages/dd-trace/src/profiling/exporters/event_serializer.js

```diff
@@ -0,0 +1,76 @@
+const os = require('os')
+const perf = require('perf_hooks').performance
+const version = require('../../../../../package.json').version
+
+class EventSerializer {
+  constructor ({ env, host, service, version, libraryInjected, activation } = {}) {
+    this._env = env
+    this._host = host
+    this._service = service
+    this._appVersion = version
+    this._libraryInjected = !!libraryInjected
+    this._activation = activation || 'unknown'
+  }
+
+  typeToFile (type) {
+    return `${type}.pprof`
+  }
+
+  getEventJSON ({ profiles, start, end, tags = {}, endpointCounts }) {
+    return JSON.stringify({
+      attachments: Object.keys(profiles).map(t => this.typeToFile(t)),
+      start: start && start.toISOString(),
```

Comment:
This is strange to check for `start` being defined.
I guess that's for the file exporter case, but maybe we should also ensure to always pass a `start` to file export.

Reply:
yeah, it's for the file exporter. Fixed.